content/blog/using-structured-outputs-in-vllm.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Using structured outputs in vLLM
+date: 2025-03-16T19:28:00.657Z
+author: Ismael Delgado Muñoz
+authorimage: /img/Avatar6.svg
+thumbnailimage: ""
+disable: false
+tags:
+  - AI
+  - GenAI
+  - opensource
+  - LLM
+---
+Generating predictable and reliable outputs from large language models (LLMs) can be challenging, especially when those outputs need to integrate seamlessly with downstream systems. Structured outputs solve this problem by enforcing specific formats, such as JSON, regex patterns, or even grammars. vLLM supported this since some time ago, but there were no documentation on how to use it, and that´s why I decided to do a contribution and write the [Structured Outputs documentation page](https://docs.vllm.ai/en/latest/usage/structured_outputs.html).
```

Comment:
At the end of the introductory paragraph, I would recommend to add a short sentence to ensure you grab the reader’s attention.

Add a line space, and you can add something like that:

In this blog post, I'll explain....

Dale Rensing, our editorial consultant will probably help you with the sentence to grab readers attention.